### DISCUSSION TITLE
Can't download beatmaps in playlists by clicking on the download beatmap button

### DISCUSSION THREAD

**Discussion by bytenekos** [ppy/osu#26403] [Q&A] [answered]
Was just wondering, are you supposed to be able to download beatmaps by just pressing the download button without selecting the beatmap first?

https://github.com/ppy/osu/assets/66184829/9bb87c85-c1cb-4814-ba23-3a19c52ebb16

I would've assumed that you could just press download instead of first selecting the beatmap then pressing download

**Comment by peppy** [comment]
Should probably be the case, yes.

**Reply by peppy** [reply, marked as answer]
#26449
